api/lib/opentelemetry/trace/event.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Trace
+    # A text annotation with a set of attributes.
+    class Event
+      EMPTY_ATTRIBUTES = {}.freeze
```

Comment:
Does this need to be public?

Reply:
It definitely doesn't. We had the same in the `Link` class. I went ahead and changed the visibility for both of them.